lmdb/env.go

```diff
@@ -103,7 +103,7 @@ func (env *Env) FD() (uintptr, error) {
 	if err != nil {
 		return 0, err
 	}
-	if mf == C.mdb_filehandle_t(-1) {
+	if uintptr(mf) == ^uintptr(0) {
```

Comment:
The RHS probably deserves a locally scoped named constant with a comment about its value.